src/refract/ElementFwd.h

```diff
@@ -2,37 +2,52 @@
 //  refract/ElementFwd.h
 //  librefract
 //
-//  Created by Jiri Kratochvil on 01/01/2016
-//  Copyright (c) 2016 Apiary Inc. All rights reserved.
+//  Created by Thomas Jandecka on 04/09/2017
+//  Copyright (c) 2017 Apiary Inc. All rights reserved.
 //
 
 #ifndef REFRACT_ELEMENTFWD_H
 #define REFRACT_ELEMENTFWD_H
 
 namespace refract
 {
-
-    // Forward declarations of Elements
-    struct IElement;
-
-    struct NullElement;
-
-    struct StringElement;
-    struct NumberElement;
-    struct BooleanElement;
-
-    struct HolderElement;
-    struct MemberElement;
-
-    struct ArrayElement;
-    struct EnumElement;
-    struct ObjectElement;
-
-    struct RefElement;
-    struct ExtendElement;
-
-    struct OptionElement;
-    struct SelectElement;
+    namespace data
```

Comment:
maybe rename to `dsd`, it is presented with given name in comments, why not use it in namespace?